src/lib/utils/hooks/SynapseAPI/forum/useForum.ts

```diff
@@ -203,7 +205,7 @@ export function useGetRepliesInfinite(
     PaginatedResults<DiscussionReplyBundle>,
     SynapseClientError
   >(
-    ['forumthread', 'infinite', threadId, limit, filter, sort, ascending],
+    ['thread', 'infinite', threadId, limit, filter, sort, ascending],
```

Comment:
If you use the thread ID second here...
```suggestion
    ['thread', threadId, 'infinite', limit, filter, sort, ascending],
```